Code/Tools/SceneAPI/SceneBuilder/Importers/AssImpSkinWeightsImporter.cpp

```diff
@@ -97,6 +97,8 @@ namespace AZ
                         }
                         Pending pending;
                         pending.m_bone = bone;
+                        pending.m_sanitizedName = bone->mName.C_Str();
+                        RenamedNodesMap::SanitizeNodeName(pending.m_sanitizedName, context.m_scene.GetGraph(), context.m_currentGraphPosition);
```

Comment:
suggestion: Change looks good but I'm wondering is there anyway we can test this? (even if we were to just extract this bit of code and pass in an unsanitized name and then check we get a correctly sanitized name out the other side?). Would be great to reference this PR when the problematic asset is merged into Automated Testing for regression testing. Thanks!

Reply:
If this is fixed then issue (#9843) can be closed.
Also I noticed before and it sounds silly but many repeated models in one fbx can cause the parser trouble with R/W bottleneck.